docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -156,13 +156,20 @@
                         others' checkpoints, and perhaps interfere with each other in other ways.
                     </td>
                 </tr>
-                <tr>
                     <td class="property" id="job-coordinator-system">job.coordinator.system</td>
                     <td class="default"></td>
                     <td class="description">
                         <strong>Required:</strong> The <span class="system">system-name</span> to use for creating and maintaining the <a href="../container/coordinator-stream.html">Coordinator Stream</a>.
                     </td>
                 </tr>
+                <tr>
+                    <td class="property" id="job-default-system">job.default.system</td>
```

Comment:
> Well, imo, exactly-once is a "property" of the producer. So, I would expect a configuration like this:
> systems.kafka-exactly.samza.factory = org.apache.pkg.KafkaSystemFactory
> systems.kafka-exactly.samza.producer.exactly-once.enabled = true

That's an even more simple scenario. It will be EXACTLY the same in the new build. We're only separating out stream configs, not system configs. 

Anything in the old config that wasn't prefixed by "systems.systemName.streams.streamsName." will have 0 changes after this patch.

> Another question : Based on your example, if I specify a producer property for a stream A and not stream B (which belongs to the same system), will we create 2 producer instances or 1 ?
Invalid scenario. We don't define producers on a per stream basis. We do it on a per system basis.